packages/filesystem/src/browser/file-dialog/file-dialog.ts

```diff
@@ -245,6 +247,22 @@ export class OpenFileDialog extends FileDialog<MaybeArray<FileStatNode>> {
         if (props.canSelectFiles !== undefined) {
             this.widget.disableFileSelection = !props.canSelectFiles;
         }
+        this.openButton = this.createButton('open single text');
+        this.controlPanel.appendChild(this.openButton);
+        this.openButton.classList.add('main');
+        this.openButton.addEventListener('click', () => this.openInputDialog());
+    }
+
+    private openInputDialog(): void {
```

Comment:
is this a leftover?

Reply:
It's just for test purposes from what I understand.
The commit `in order to test` will be removed in the final version.

Reply:
the test code is commited by me